ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplLoadTask.java

```diff
@@ -471,6 +472,7 @@ private int executeIncrementalLoad(DriverContext driverContext) {
         if (work.hasBootstrapLoadTasks()) {
           LOG.debug("Current incremental dump have tables to be bootstrapped. Switching to bootstrap "
                   + "mode after applying all events.");
+          work.setBootstrapDuringIncLoad(true);
```

Comment:
We shall set it in ReplLoadWork constructor once for all.

Reply:
done